### PR TITLE
Bug fixes and trimming.

### PR DESCRIPTION
**Pull request by bjornborg**
Fixed bugs in the scripts which made it skip pacman stage.

Altered logic to create a symbolic link to sdcard even if there is no install-user file.

Deleted source configuration line in adc and i2c setups since they are not used.

### CODE REVIEW

File: arm-bbb/setup-available/setup-post-01-pru.sh [hunk 1]
```diff
@@ -1,93 +0,0 @@
-#!/bin/bash
```
Comment (olbender) [inline arm-bbb/setup-available/setup-post-01-pru.sh]: Why is this file deleted?

**Review by olbender — COMMENTED** [review]
See question about pru setup.

File: arm-bbb/setup-available/setup-post-01-pru.sh (hunk 1 above)
Comment (bjornborg) [reply arm-bbb/setup-available/setup-post-01-pru.sh]: I thought it was not needed anymore at the time. I guess I was wrong

**Review by olbender — APPROVED** [review]
